### PR TITLE
Improving the README.md with overview and other resources

### PR DESCRIPTION
**Pull request by vidya-mohan**
**Issue #:** 
<No Issue> 
**Description of changes:**
Improving README.md by fixing typo and adding additional resourcing details.

**Testing**
1. Have you successfully run `npm run build:release` locally?
No
2. How did you test these changes? 
Preview via Github
3. If you made changes to the component library, have you provided corresponding documentation changes?
Not applicable
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,67 @@
-# Chime SDK UI Component Library
+# Chime SDK React Components Library
```
Comment (devalevenkatesh) [inline README.md:1]: Should we start with Amazon here as we do in the JS SDK README?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,67 @@
...
+<a href="https://www.npmjs.com/package/amazon-chime-sdk-component-library-react"><img src="https://img.shields.io/npm/v/amazon-chime-sdk-component-library-react?style=flat-square"></a>
+<a href="https://github.com/aws/amazon-chime-sdk-component-library-react"><img src="https://github.com/aws/amazon-chime-sdk-component-library-react/workflows/CI%20Workflow/badge.svg"></a>
+
+[React Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/)
+[Chime SDK JavaScript Library](https://github.com/aws/amazon-chime-sdk-js/)
```
Comment (devalevenkatesh) [inline README.md:6]: Should this say Amazon Chime SDK Javascript Library?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,67 @@
...
+
+The Amazon Chime SDK makes it easy to add collaborative audio calling, video calling, and screen share features to web applications by using the same infrastructure services that power millions of Amazon Chime online meetings.
+
+The Amazon Chime SDK React Component Library supplies client-side state management and reusable UI components for common web interfaces used in audio and video conferencing applications, including: video tile grids, microphone activity indicators, and call controls. All components come with a simple, modern design, and can be used as-is or restyled with a custom theme. In addition to UI components, the library leverages Reacts' state management tools such as Providers and Hooks to connect to the Amazon Chime SDK and pass data to the UI layer, simplifying state synchronization so that developers can concentrate on building engaging experiences.
```
Comment (devalevenkatesh) [inline README.md:10]: Should we mention specifically "Amazon Chime SDK for Javascript" in the last line?

For e.g: .... Amazon Chime SDK for Javascript and pass data to the UI layer, ..

File: README.md [hunk 4]
```diff
@@ -1,34 +1,67 @@
...
+[Amazon Chime SDK for JavaScript Documentation](https://aws.github.io/amazon-chime-sdk-js/#amazon-chime-sdk-for-javascript)
 
-[Chime SDK UI Component Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/)
 [Migration to V2](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/migration-to-v2--page)
 
-<a href="https://www.npmjs.com/package/amazon-chime-sdk-component-library-react"><img src="https://img.shields.io/npm/v/amazon-chime-sdk-component-library-react?style=flat-square"></a>
-<a href="https://github.com/aws/amazon-chime-sdk-component-library-react"><img src="https://github.com/aws/amazon-chime-sdk-component-library-react/workflows/CI%20Workflow/badge.svg"></a>
+## Examples
+
+[Chime SDK Meeting Demo](https://github.com/aws/amazon-chime-sdk-component-library-react/tree/master/demo/meeting)
```
Comment (devalevenkatesh) [inline README.md:32]: Should this be titled "Amazon Chime SDK React Meeting Demo" ?

File: README.md [hunk 5]
```diff
@@ -1,34 +1,67 @@
...
+
+[Chat Demo](https://github.com/aws/amazon-chime-sdk-component-library-react/tree/master/demo/chat)
+
+## Installation and Development
 
-This contains reusable components written with React, TypeScript and styled components to be used to create UIs with Chime SDK JS.
+If you are adding this library to your existing application, add amazon-chime-sdk-component-library-react and the necessary peer dependencies to your project
```
Comment (devalevenkatesh) [inline README.md:38]: Nit: You can use backtick for the package name. For e.g: `amazon-chime-sdk-component-library-react`.

Comment (devalevenkatesh) [reply README.md:38]: Nit: Need a "." at the end of sentence here.

File: README.md [hunk 6]
```diff
@@ -1,34 +1,67 @@
...
+
+```
+npm install --save amazon-chime-sdk-component-library-react amazon-chime-sdk-js styled-components styled-system 
+```
 
-## To genereate dependencies
+If you have cloned this repo, then execute
```
Comment (devalevenkatesh) [inline README.md:44]: Would it help to separate this into two commands?

### Clone this repo
```
git clone https://github.com/aws/amazon-chime-sdk-component-library-react.git
```

### Generate dependencies
```
npm install
```

**Review by devalevenkatesh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vidya-mohan) [reply README.md:1]: yeah good point

**Comment by devalevenkatesh** [comment]
Thanks for taking this up and improving the README 👍

File: README.md (hunk 6 above)
Comment (vidya-mohan) [reply README.md:44]: i merged them into one

File: README.md (hunk 4 above)
Comment (vidya-mohan) [reply README.md:32]: actually i reverted that - its seems like an overkill to saw Amazon Chime SDK here -

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**